build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
+                -DBUILD_PKGCONFIG_FILES=ON \
+                -DBUILD_DOC=ON \
+                -DZLIB_DLL=ON \
+                -D_WIN32=ON
```

Comment:
I'm going to see if I can track down the build failure with Autotools.  Until then, I think this workaround is fine.